integration_tests/features/challenge_double_spending_flow.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Challenge double spending flow
+
+  Scenario: userA deposits to plasma cash. userA and transfers to userB first and then transfers the same funds to userC. userC tries to withdraw the funds and userB challenges the exit.
+    Given userA has 100 eth in root chain
```

Comment:
I don't think we need this line here anymore. 
Meanwhile, I'm thinking probably we could change description a bit:

```
Given userA deposits 1 eth to plasma cash and get a 1 eth coin
and userA transfer that coin to userB
and userA double spend that coin to userC
When userC tries to withdraw the coin
Then root chain get the start-exit record
When userB challenges the withdraw
Then the challenge is successful
```

I don't think other places should follow the style in `basic_flow`. In basic flow the test tries to test a bunch of things in one goal to provide basic end to end test. However, other tests can be easier and don't need to reply the test (verifying deposit & transfer) already done there. And in general, I think we should make what our focus on the test clearer.